etc/docker/dev/rucio.cfg

```diff
@@ -184,3 +184,10 @@ scope = user.vzavrtan
 
 [permission]
 policy=atlas
+
+[policy]
+permission = atlas
+schema = atlas
+lfn2pfn_algorithm_default = hash
+support = https://github.com/rucio/rucio/issues/new
```

Comment:
for atlas, the default here should be

support = hn-atlas-dist-analysis-help@cern.ch

and i would take out the /new at the end of support_rucio. showing the list of open issues can prevent duplicate submissions

Reply:
It's for the docker dev environment. Having github at both places is fine imo.